src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -471,6 +471,10 @@ public enum CassandraRelevantProperties
     // Allow restoring legacy behavior of deleting sai components before a rebuild (which implies a rebuild cannot be
     // done without first stopping reads on that index)
     IMMUTABLE_SAI_COMPONENTS("cassandra.sai.immutable_components", "false"),
```

Comment:
Side note: This comment is not very clear explaining whether the property should be set to true or false to restore the legacy behavior. I believe if the property is `false` then the behavior is legacy because ISC are disabled, correct?